terraform/20-app/locals.tf

```diff
@@ -26,6 +26,7 @@ locals {
   enable_public_db                             = local.is_dev
   is_dev                                       = var.environment_type == "dev"
   is_prod                                      = local.environment == "prod"
+  is_ready_for_etl                             = contains(["dev", "test"], var.environment_type)
```

Comment:
This is the temp var which allows us to conditionally add the IAM policy for cross account permissions.
Once we've got the uat and prod accounts, we can delete this